src/test/scala/temple/generate/kube/UnitTestData.scala

```diff
@@ -1,12 +1,13 @@
 package temple.generate.kube
 
 import temple.generate.kube.ast.OrchestrationType._
+import temple.utils.FileUtils
 
 object UnitTestData {
 
   val basicOrchestrationRoot: OrchestrationRoot = OrchestrationRoot(
     Seq(
-      Service("user", "temple-user-service", Seq(80)),
+      Service(name = "user", image = "temple-user-service", ports = Seq(80), replicas = 1, secretName = "regcred"),
```

Comment:
What’s the secret name for?

Reply:
The secret stores the credentials for kube to access the docker registry - it's not created in the orchestration script for security reasons but it still needs to know the name of the credential so it can look it up in the kube system